app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    def __init__(
+        self,
+        name: str,
+        appetite: int,
+        is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> None:
+        food_points = 0
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            food_points += self.appetite
+        return food_points
```

Comment:
"set is_hungry to False and return number of eaten food points if animal is hungry. Otherwise, it should return 0 and print nothing"
You dont need food_points variable, you just need to return self.appetite if animal is hungry, else return 0